requirements/testing/travis36.txt

```diff
@@ -1,6 +1,6 @@
 # Extra pip requirements for the travis python 3.6 build
 
 ipykernel
-nbconvert[execute]
+nbconvert[execute]!=5.0.0
```

Comment:
I think it's nbformat you want to !=5.0.0, nbconvert is already at 5.6.1.

Reply:
Yup